README.md

```diff
@@ -42,7 +42,7 @@ You then need to load the plugin. In `boostrap.php`, something like:
 For a complete setup, add the following to your `App\Controller\AppController`:
 
 ```php
-public $components = ['BootstrapUI.Flash'];
+public $theme = 'BootstrapUI';
```

Comment:
To be honest, I find this plugin confusing.

It's sort of a theme, but you don't use it as a theme. I've reverted removing the flash component, addressing one of the _other_ things I found confusing (the use of symlinks, unconventional flash element names, and forcing the use of elements in the plugin that don't exist (if a user were to do `$this->Flash->foo('message')).